apps/studio/components/interfaces/Auth/Users/UsersV2.tsx

```diff
@@ -484,7 +498,6 @@ export const UsersV2 = () => {
                     return (
                       <Row
                         {...props}
-                        key={props.row.id}
```

Comment:
this key prop was unnecessary as its handled in ...props already
in fact it was causing some rendering issues when there's a lot of data loaded, and you search in the input field as such:

![image](https://github.com/user-attachments/assets/ce9e3175-1d45-4a57-9a14-bc9ff76d2c83)